item.rb

```diff
@@ -0,0 +1,26 @@
+require 'date'
+
+class Item
+  attr_reader :id
+  attr_accessor :archived, :genre, :publish_date, :author
+
+  def initialize(publish_date, archived: false)
+    @id = Random.rand(1..1000)
+    @publish_date = publish_date
+    @archived = archived
+  end
+
+  def can_be_archived?
+    year = Time.new.year
+    year - @publish_date.year > 10
+  end
+
+  def move_to_archive
+    self.archived = can_be_archived?
+  end
+
```

Comment:
- [ ] As required for the project, I noticed that the `move_to_archive()` method in the Item class has to reuse `can_be_archived?()` method, to change the archived property to true if the result of the` can_be_archived?()` method is true, and do nothing if the result of the `can_be_archived?()` method is false. But in your case, I can't see all the conditions be implemented. I think this should be updated to something like this:
```ruby
   def move_to_archive
      if can_be_archived?
        @archived = true
      end
    end
```